### PR TITLE
Add fastpca tool

### PR DESCRIPTION
**Pull request by simonbray**
Tool for PCA of MD simulations. Integrating for the physics people.

### CODE REVIEW

File: tools/biomoldyn/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: biomoldyn
+owner: chemteam
+description: "Tools developed by the Biomolecular Dynamics group at the University of Freiburg"
```
Comment (bgruening) [inline tools/biomoldyn/.shed.yml]: it should describe the tool not the author

File: tools/biomoldyn/fastpca.xml [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+            #else:
+                -S '$inputs.stats'
+            #end if
+            -l '$output_val'
+            $norm
+            ##-b "\${GALAXY_MEMORY_MB:-4096}"
+            $periodic
+            $dynamic_shift
+            --verbose
+            ##-n "\${GALAXY_SLOTS:-1}"
```
Comment (bgruening) [inline tools/biomoldyn/fastpca.xml]: ?

File: tools/biomoldyn/fastpca.xml [hunk 3]
```diff
@@ -0,0 +1,129 @@
+<tool id="fastpca" name="fastpca" version="@VERSION@">
+    <description>- dimensionality reduction of MD simulations</description>
```
Comment (bgruening) [inline tools/biomoldyn/fastpca.xml]: I would remove the `- `

**Comment by simonbray** [comment]
thanks Björn!

**Pull request merged**